src/com/projectkorra/projectkorra/Element.java

```diff
@@ -57,10 +57,11 @@ public String getBend() {
 	public static final SubElement SAND = new SubElement("Sand", EARTH);
 	public static final SubElement LIGHTNING = new SubElement("Lightning", FIRE);
 	public static final SubElement COMBUSTION = new SubElement("Combustion", FIRE);
+	public static final SubElement BLUE_FIRE = new SubElement("Blue_Fire", FIRE);

```

Comment:
`Blue_Fire` underscore bad nightmare bad bad character

Reply:
No better option ma dude, unless you want  a hyphen?

Reply:
it's called a space

Reply:
I don't want to mess w/ the Add Command and have to rewrite that part to work with a space.

Reply:
You already had to add stuff to make an exception for the underscore because obviously we don't want that showing up in any of the text the plugin sends to a player, having a space and then just ignoring it for most purposes is easier.

Reply:
Personally, I think using camelcase would be the best option to match with the way we handle ability names.